restli-server/src/main/java/com/linkedin/restli/internal/server/model/ResourceMethodDescriptor.java

```diff
@@ -434,6 +434,24 @@ public String getMethodName()
     return _type.toString();
   }
 
+  /**
+   * Returns the resource method identifier if it can be determined, or null otherwise.
+   * This is identical to Request.getResourceMethodIdentifier().
+   * @return the resource method identifier if it can be determined, or null otherwise.
+   */
+  public String getResourceMethodIdentifier() {
```

Comment:
Theoretically, this can become stale if the resource model's URI template is updated after parents/subs are added. Would it be ok in just calling the `generate` method every time? Would the performance impact be that great?

Reply:
The scan for properties isn't free. Do you think either the ResourceModel or the ResourceModel.baseUriTemplate changes on the fly?